version_align.sh

```
@@ -0,0 +1,49 @@
+
+# GNU/BSD compat
+sedi=(-i'')
+case "$(uname)" in
+  # For macOS, use two parameters
+  Darwin*) sedi=(-i '')
+esac
+
+echo "Updating ..."
+
+# Update the version for all crates in the Cargo.toml workspace.dependencies section
+sed "${sedi[@]}" -e '/\[workspace.dependencies\]/,/# Magicblock/s/version = ".*"/version = "='$version'"/' Cargo.toml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)